src/vm/castcache.h

```diff
@@ -0,0 +1,167 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// File: castcache.h
+//
+
+#ifndef _CAST_CACHE_H
+#define _CAST_CACHE_H
+
+#include "util.hpp"
+
+struct CastHashEntry
+{   
+    TypeHandle          sourceType;
+    TypeHandle          targetType;
+    BOOL                result;
+
+    DWORD               version1;
+    DWORD               version2;
+};
+
+class CastCache
+{
+public:
+    CastCache(DWORD size)
+    {
+        m_TableSize = size;
+        m_Table = new CastHashEntry[size];
+        memset(m_Table, 0, size);
```

Comment:
Shouldn't `size * sizeof(CastHashEntry*)` or something similar be used here?

Reply:
Yes, thanks!!